l10n/bundle.l10n.de.json

```diff
@@ -1,100 +1,101 @@
+  "Method not implemented.": "Die Methode ist nicht implementiert.",
   "Missing connectionType in active profile.": "Fehlender Verbindungstyp im aktiven Profil.",
   "Missing endpoint in active profile.": "Fehlender Endpunkt im aktiven Profil.",
   "Missing host in active profile.": "Fehlender Host im aktiven Profil.",
   "Missing port in active profile.": "Fehlender Port im aktiven Profil.",
   "Missing sas path in active profile.": "Fehlender SAS Pfad im aktiven Profil.",
   "Missing username in active profile.": "Fehlender Benutzername im aktiven Profil.",
-  "Switch Current SAS Profile": "Aktuelles SAS Profil wechseln",
-  "Select a SAS connection profile": "Wählen Sie ein SAS Verbindungsprofil",
+  "New File": "Neue Datei",
+  "New Folder": "Neuer Ordner",
   "New SAS Connection Profile Name": "Neuer SAS Verbindungsprofilname",
-  "Enter connection name": "Geben Sie einen Verbindungsprofilname ein",
-  "You can also specify connection profile using the settings.json file.": "Sie können ein Verbindungsprofil auch über die settings.json Datei eingeben.",
-  "SAS Viya Server": "SAS Viya Server",
-  "Enter the URL": "Geben Sie die URL ein",
-  "Enter the URL for the SAS Viya server. An example is https://example.sas.com.": "Geben Sie die URL des SAS Viya Server ein. Hier ein Beispiel https://example.sas.com",
+  "No Active Profile": "Kein aktives Profil",
+  "No Profile": "Kein Profil",
+  "No Profiles available to delete": "Es gibt keine Profile die gelöscht werden können",
+  "No SAS Connection Profile": "Kein SAS Verbindungsprofil",
+  "No authorization code": "Kein Authorisierungs Code",
+  "Not implemented": "Nicht implementiert",
+  "Paste authorization code here": "Fügen Sie hier Ihren Authorisierungscode ein",
+  "Port Number": "Port",
+  "Rename File": "Datei umbennen",
+  "Rename Folder": "Ordner umbenennen",
+  "Result": "Ergebnis",
+  "SAS 9 SSH Server": "SAS 9 SSH Server",
   "SAS Compute Context": "SAS Compute Context",
-  "Enter the SAS compute context": "Geben Sie den SAS Compute Context ein",
-  "Enter the SAS compute context.": "Geben Sie den SAS Compute Context ein.",
-  "Client ID": "Client ID",
-  "Enter a client ID": "Geben Sie eine Client ID ein",
-  "Enter the registered client ID. An example is myapp.client.": "Geben Sie die registrierte Client ID ein. Hier ein Beispiel myapp.client.",
-  "Client Secret": "Client Secret",
-  "Enter a client secret": "Geben Sie ein Client Secret ein",
-  "Enter secret for client ID. An example is myapp.secret.": "Geben Sie das Secret für die Client ID ein. Hier ein Beispiel myapp.secret.",
-  "Connection Type": "Verbindungstyp",
+  "SAS Log": "SAS Log",
+  "SAS Server Username": "SAS Server Benutzername",
+  "SAS Viya Server": "SAS Viya Server",
+  "SAS code running...": "SAS Code wird ausgeführt...",
+  "SSH_AUTH_SOCK not set. Check Environment Variables.": "SSH_AUTH_SOCK wurde nicht gesetzt. Bitte die Umgebungsvariablen überprüfen.",
+  "Saving {itemName}.": "Saving {itemName}.",
```

Comment:
Hey @Criptic. Would you mind providing me with a translation of this key?

Reply:
"SAS Viya Server": "SAS Viya Server",
  "SAS code running...": "SAS Code wird ausgeführt...",
  "SSH_AUTH_SOCK not set. Check Environment Variables.": "SSH_AUTH_SOCK wurde nicht gesetzt. Bitte die Umgebungsvariablen überprüfen.",
  "Saving {itemName}.": "Speichere {itemName}.",

Reply:
Thanks!